edx_proctoring/views.py

```diff
@@ -126,6 +126,16 @@ def wrapped(request, *args, **kwargs):  # pylint: disable=missing-docstring
     return wrapped
 
 
+def is_user_course_or_global_staff(user, course_id):
+    """
+        Return whether a user is course staff for a given course, described the course_id,
+        or is global staff.
+    """
+    instructor_service = get_runtime_service('instructor')
+
+    return not user.is_staff or not instructor_service.is_course_staff(user, course_id)
```

Comment:
Why `not`? This returns the opposite of what you want.

Reply:
I forgot to push up the fix; thanks.